### DISCUSSION TITLE
JavaScript VM

### DISCUSSION THREAD

**Discussion by talhazullfiqar** [Cyfrin/foundry-full-course-cu#2575] [Q&A]
Hey,

I am currently at the FundMe section of the Solidity Smart Contract Development (Beginner) course. The instructor is using the JavaScript VM in the Remix deployer, but there's no option to select the JavaScript VM in my deploy section. Did I miss something?

![Screenshot 2024-09-29 162317](https://github.com/user-attachments/assets/d6f02036-b63b-4fc4-82b1-db241d7e1401)

**Comment by EngrPips** [comment]
Things might have changed from when the Instructor filmed the course, so please use `Remix Vm (Cancun)` instead.

**Comment by abishek1740P** [comment]
Ensure You're Using Remix IDE:
Go to the Remix IDE in your browser.

Check the Environment Dropdown:
In the "Deploy & Run Transactions" panel on the right-hand side of the Remix IDE, you should see an "Environment" dropdown menu.

Make sure it's expanded, and you should see options like:
JavaScript VM (Berlin)
JavaScript VM (London)
Injected Web3
Web3 Provider
If "JavaScript VM" is not visible, you may be in a different mode or section of Remix.

Switch to the Correct Plugin:
Ensure that you are in the correct Solidity plugin. You can check this in the left sidebar under the File Explorer and Solidity Compiler sections.

Clear Browser Cache:
If Remix IDE is misbehaving, sometimes clearing your browser’s cache or refreshing the Remix page solves the issue.

Disable Extensions:
Some browser extensions might interfere with the Remix IDE. Try disabling any ad blockers or privacy extensions.

**Comment by EngrPips** [comment]
Thanks for the detailed heads-up.
